### ISSUE TITLE
Validation loss keeps going up when training using a pre-trained model

### ISSUE THREAD

**Issue by serg06** [NVIDIA/flowtron#97]
I'm trying to train using pre-trained `libritts2p3k` model on my own data:

```
python train.py
  -c config.json
  -p
    train_config.ignore_layers=["speaker_embedding.weight"]
    train_config.checkpoint_path="models/flowtron_libritts2p3k.pt"
    data_config.training_files="..."
    data_config.validation_files="..."
```

but my validation loss keeps going up for some reason:

![image](https://user-images.githubusercontent.com/1350889/102663898-b3d16b00-414f-11eb-9e8e-5c6474ba41a1.png)

Yet when I try to fine tune it instead:

```
python train.py
  -c config.json
  -p
    train_config.finetune_layers=["speaker_embedding.weight"]
    train_config.checkpoint_path="models/flowtron_libritts2p3k.pt"
    data_config.training_files="..."
    data_config.validation_files="..."
```

the validation loss goes down as expected:

![image](https://user-images.githubusercontent.com/1350889/102664075-0dd23080-4150-11eb-88af-a5e024553d2e.png)

Not sure why... any ideas?

**Comment by rafaelvalle** [comment]
I assume your dataset is small and the model is overfitting to the training data, which results in bad performance for validation data.
Fine-tuning in this repo only modifies the speaker embedding while holding other model parameters fixed.
In the fine-tuning setup is much harder to overfit the training data.

**Comment by serg06** [comment]
I see, thanks
